scio-extra/src/it/scala/com/spotify/scio/extra/sparkey/SparkeyIT.scala

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2017 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.scio.extra.sparkey
+
+import java.nio.ByteBuffer
+
+import com.spotify.scio.testing.PipelineSpec
+import org.apache.beam.sdk.options.{GcpOptions, GcsOptions}
+import org.apache.beam.sdk.util.GcsUtil.GcsUtilFactory
+import org.apache.beam.sdk.util.gcsfs.GcsPath
+
+import scala.collection.JavaConverters._
+
+class SparkeyIT extends PipelineSpec {
+  import com.spotify.scio.extra.sparkey._
+
+  val sideData = Seq(("a", "1"), ("b", "2"), ("c", "3"))
+
+  "SCollection" should "support .asSparkeySideInput using GCS tempLocation" in {
+    runWithContext { sc =>
+      val tempLocation = sc.optionsAs[GcpOptions].getGcpTempLocation
+      sc.options.setTempLocation(tempLocation)
```

Comment:
Why is this necessary? Because `GcpOptions` creates a temp bucket silently but doesn't set `tempLocation`?